features/advocate_new_claim.feature

```diff
@@ -181,3 +181,24 @@ Feature: Advocate new claim
       And I submit to LAA
      Then There should not be any Initial Fees saved
       And There should be a Miscellaneous Fee Saved
+
+  @javascript @webmock_allow_localhost_connect
```

Comment:
I don't think you need @webmock_allow_localhost_connect, as far as I understand it this is just to allow ajax requests to be used in the tests and this feature doesn't require any ajax. I've removed it locally and tested and the tests still pass.